packages/dashboards/examples/src/components/docs/widget-types/proportional/proportional-docs.component.html

```diff
@@ -24,6 +24,22 @@ <h1>Proportional Widget</h1>
     <proportional-widget-example></proportional-widget-example>
 </nui-example-wrapper>
 
+<h3>Interaction Handling</h3>
+<p>
+    In the example below the first chart is using the <code><a href="../injectables/UrlInteractionHandler.html" target="_blank">UrlInteractionHandler</a></code> to handle interactions that occur on the legend when the chart is set to interactive.
+    The second chart shows how individual series can be interactive when the series data, that conform to <code><a href="../interface/IProportionalWidgetData.html" target="_blank">IProportionalWidgetData</a></code> interface, and provide value for:
```

Comment:
There should be probably <a href="../interface**s**/IProportionalWidgetData.html" target="_blank"> instead of _interface_